### DISCUSSION TITLE
Set Waydroid to start on PC startup

### DISCUSSION THREAD

**Discussion by AdrianKrajnik2** [Vanilla-OS/live-iso#388] [Q&A]
It is possible to set Waydroid to start on PC startup. It is very annoying to wait when I want to launch an Android app.

**Comment by taukakao** [comment]
How are you even using waydroid? Last time I checked it pretty much doesn't work at all.
If you are unsure if your system is up to date run `abroot upgrade --check-only`.

To answer your question, we currently don't have enough devlopers to make a proper waydroid implementation. We are currently focusing on improving existing stable features and updates.

**Reply by AdrianKrajnik** [reply]
I initialized waydroid with the command: vso android init. Waydroid initialized, but it didn't work at first. I tried to initialize again. Then the command line said that waydroid is already initialized and an option that I can force waydroid to reinitialize. After this initialization I was able to install apk files using the command line, but waydroid still has a few bugs. 

and this is my another account
